### DISCUSSION TITLE
_next folder not found

### DISCUSSION THREAD

**Discussion by ay0ks** [vercel/next.js#33685] [Help] [answered]
### Run `next info` (available from version 12.0.8 and up)

    Operating System:
      Platform: linux                                                                                                                                
      Arch: x64                                                                                                                                      
      Version: #102-Ubuntu SMP Fri Nov 5 16:31:28 UTC 2021                                                                                           
    Binaries:                                                                                                                                        
      Node: 16.13.2                                                                                                                                  
      npm: 8.1.2                                                                                                                                     
      Yarn: 1.22.17                                                                                                                                  
      pnpm: 6.28.0                                                                                                                                   
    Relevant packages:                                                                                                                               
      next: 12.0.8                                                                                                                                   
      react: 17.0.2                                                                                                                                  
      react-dom: 17.0.2                                                                                                                              

### What version of Next.js are you using?

12.0.8

### What version of Node.js are you using?

16.13.2

### What browser are you using?

Firefox

### What operating system are you using?

Ubuntu

### How are you deploying your application?

yarn dev

### Describe the Bug

After trying to access website, devtool's console shows an errors that say that folder _next and all files in it are not accessible (there's only .next) folder
![изображение](https://user-images.githubusercontent.com/43708417/151139089-7ccda638-5b2c-4dc3-bb54-2072ed9f96e6.png)

### Expected Behavior

Application must work normally.

### To Reproduce

Create project with npx create-next-app

**Comment by ay0ks** [comment]
Also tried to proxy_pass folder using nginx, but no result

**Comment by ay0ks** [comment]
After changing distDir, everything works normally.

**Reply by theropoy** [reply, marked as answer]
Hi, did you just change distDir? I got currently the same problem and after changing distDir it still tries to import from _next

**Reply by ay0ks** [reply]
> Hi, did you just change distDir? I got currently the same problem and after changing distDir it still tries to import from _next

Hello and yes, changing distDir helped for me, I don't know about other solutions though.
